src/conspiracies/extract_ents_heads/functions.py

```diff
@@ -0,0 +1,77 @@
+import spacy
+from collections import Counter
+
+
+def get_headword(noun_phrases: list) -> list:
+  """Extracts headwords with their entity labels from noun phrases
+   Args:
+       noun_phrases (list): A list of lists with strings (noun phrases) 
+   Returns:
+       headwords (list): A list of strings containing headwords and their entity types
+  """ 
```

Comment:
I assume it is a list of spans not a list of strings?

```suggestion
def get_headword(noun_phrases: List[Span]) -> List[str]:
  """Extracts headwords with their entity labels from noun phrases
   Args:
       noun_phrases (List[Span]): A list of lists with spans (noun phrases) 
   Returns:
       List[str]: A list of strings containing headwords and their entity types
  """ 
```

for this, you will need to add
```
from typing import List
from spacy.tokens import Span
```